### PR TITLE
Swag opportunity for Scalyr Trial

### PR DESCRIPTION
**Pull request by robincher**
- [x] I've checked that there isn't a duplicate pull request.

**Is it regarding a new swag proposal or something else?**  

This is a new swag proposal

**Q. Have you received swag from this?**
Delivery in progress
**Q. How long did it take?**
TBC
**Q. Does it ship worldwide?**
Yes

### CODE REVIEW

**Comment by robincher** [comment]
Yeah, will get back here when they contacted me   :)

For those that prefer to not look into the markdown file, here is the link for you to sign-up
https://www.scalyr.com/signup

To be eligible for the swag, you gotta install their scalyr agent in a host (docker is supported) and set up at least 1 log parser to aggregate the logs, and have those logs uploaded to their end.

**Comment by EndermanOfCoding** [comment]
@rahulkumaran can you tell me how to do that? any article or doc link?

**Comment by robincher** [comment]
@swapagarwal  yo, can label this as hacktoberfest event :)

**Comment by robincher** [comment]
@swapagarwal  hehe yeah i just read through the whole terms, seems like we don't have to label it anymore ? :)

**Comment by andreathniah** [comment]
Is there anything we should do after setting up the Scalyr account and have the log data parsed? I can't seem to find any link to fill up details or any of such.

**Comment by robincher** [comment]
@andreathniah did anyone contacted you?

**Comment by agarwal-akash** [comment]
> I got a note from their representative (after some effort configuring :p ) , so shall awaits my t-shirt.
> 
> Anyway, do give their tool a try if you are embarking on building your monitoring stack.

Hey did you get an email from Courtney Dunn from Scalyr?

**Comment by andreathniah** [comment]
> @andreathniah did anyone contacted you?

@robincher Yes, I received an email from Courtney Dunn a few hours ago. Waiting for his reply at the moment.

**Comment by aslafy-z** [comment]
I just received an email from Kehau Likio, Sales Development Representative, they are gonna send a t-shirt my way !

File: README.md [hunk 1]
```diff
@@ -19,6 +18,7 @@ Sign up at [https://upscri.be/67d2fa](https://upscri.be/67d2fa) to get early acc
 - [Mullvad](#mullvad-) ![](https://img.shields.io/badge/difficulty-easy-green.svg)
 - [Netlify](#netlify-) ![](https://img.shields.io/badge/difficulty-easy-green.svg)
 - [npm](#npm-) ![](https://img.shields.io/badge/difficulty-hard-red.svg)
+- [Scalyr](#scalyr) ![](https://img.shields.io/badge/difficulty-medium-yellow.svg
```
Comment (pxgamer) [inline README.md]: Missing a closing `)` at the end.

Comment (aslafy-z) [reply README.md]: You can drop the readme changes ;)

**Comment by swapagarwal** [comment]
@robincher Please rebase this on master.

**Comment by robincher** [comment]
Let me re send a PR. think i did a wrong rebase :x

**Comment by aslafy-z** [comment]
@robincher It should be OK if you delete the two empty lines at the end of `data.json` :wink:

**Comment by arshadkazmi42** [comment]
I was giving it a try. What does this means 

> and have those logs uploaded to their end.

@robincher 

I have done with  the parser setup and can see the logs in the dashboard. Is it all? or I need to do something else?

**Comment by robincher** [comment]
> I was giving it a try. What does this means
> 
> > and have those logs uploaded to their end.
> 
> @robincher
> 
> I have done with the parser setup and can see the logs in the dashboard. Is it all? or I need to do something else?

I guess you just have to awaits their representative to contact you.

File: data.json [hunk 2]
```diff
@@ -126,5 +126,13 @@
     "reference": "https://developers.whatismybrowser.com/api/swag/",
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "tags": ["stickers"]
+  },
+  {
+    "name": "Scalyr",
+    "difficulty": "hard",
+    "description": "<a href='https://www.scalyr.com/signup'>Sign up</a> for a 30days Scalyr trial and get a free t-shirt!",
+    "reference": "https://www.scalyr.com/signup",
+    "image": "https://www.scalyr.com/assets/images/scalyr-logo-color-reverse.png",
```
Comment (aslafy-z) [inline data.json]: Here's the possible graphics of the t-shirt: https://twitter.com/scalyr/status/1040286617733345280

Maybe we can take only one of the two for illustrating?

Comment (aslafy-z) [reply data.json]: Here a cut

dm_xkkxu8aayga0 1

**Comment by arshadkazmi42** [comment]
@all I have tried this and got a mail from them saying they have discontinued this

```
Hi Arshad, 

Unfortunately we've discontinued our t-shirt campaign for the time being, so apologies for any inconveniences. 
```

**Comment by aslafy-z** [comment]
So bad, it's still live on their sign-in page: https://www.scalyr.com/signup

**Comment by arshadkazmi42** [comment]
@aslafy-z Yes. I completed everything then got to know that it is discontinued 😞

**Comment by swapagarwal** [comment]
@arshadkazmi42 I feel you! :(
@aslafy-z Merging with `active` set to `false` does not add any value IMO. I was thinking of introducing a `validUntil` key instead of `active` key as that would actually help us removing the `inactive` promotions automatically (w/o manual intervention).

**Comment by ssiddhantsharma** [comment]
@arshadkazmi42 Yeah I received a reply to the email I sent to them. The swag opportunity is closed and dusted for now.

**Pull request closed without merging**